CONTRIBUTING.md

```diff
@@ -39,9 +32,9 @@ cd my-template-ci/example;
 bk local run ../pipeline.yaml
 ```
 
-## DatoCMS
+# Publication on Buildkite.com
 
-The templates defined in this repository are ingested into DatoCMS.
+The templates defined in this repository are ingested into DatoCMS. There are two human review stages: once via PR in this repo, and another via approval in DatoCMS.
```

Comment:
Currently all merged templates are auto-published so I reckon you can remove this point.

Reply:
Oh great! I forgot to call out this for review -- it was a guess on my part!